Gemfile

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 ruby File.read('.ruby-version', mode: 'rb').chomp.prepend('~> ')
+gem 'bundler', '1.17.3'
```

Comment:
Inside the Gemfile.base is better

Reply:
And run `make bundle`

Reply:
@hallelujah i kind of like it grouped with ruby version. why is gemfile.base better?

Reply:
Because then you will need to add it to `gemfiles/prod/Gemfile` too

Reply:
right, gemfile version is also valid for production, not like ruby version. Thanks.